src/app/modules/main/chat_section/module.nim

```diff
@@ -52,7 +52,6 @@ type
     chatContentModules: OrderedTable[string, chat_content_module.AccessInterface]
     moduleLoaded: bool
     chatsLoaded: bool
-    usersModule: users_module.AccessInterface
```

Comment:
So, if I understand a NIM code correctly, this removes the logic which we want to have in [Refactor Members list to only have one list per community](https://github.com/status-im/status-desktop/issues/9090) ticket

Reply:
Before my changes user module was shared across all channels of given community. Now each channel has its own user module and that's desired, because channels may have different members.